### PR TITLE
Teams Page Added

### LINKED ISSUE #219: Team Page
**Describe the solution you'd like**
Make the Login screen as same as the _mockup_

**Contents**
1. Use Proper fonts 
2. Use already existing icons on the page.
3. For the UI elements refer to the website [www.flaticon.com](https://www.flaticon.com/).

**Please mention the work you are doing before we assign u this task.**
**Follow the UI Strictly**

### PR DESCRIPTION
**Pull request by Anushka-shukla**
Teams Page Added

### CODE REVIEW

File: Teams.dart [hunk 1]
```diff
@@ -0,0 +1,645 @@
...
+      "imgUrl": "https://dscvit.com/images/designers/shravani.jpg",
+      "name": "Shravani Shete",
+      "title": "Graphic"
+    },
+    {
+      "imgUrl": "https://dscvit.com/images/designers/ronish.jpg",
+      "name": "Ronish Sawal",
+      "title": "Graphic"
+    },
+];
```
Comment (ousnik) [inline Teams.dart]: Why do we have people from DSC VIT here?

**Comment by smaranjitghose** [comment]
@Anushka-shukla If you need assets and names for this checkout the website repo please

**Comment by Anushka-shukla** [comment]
Okay will do the necessary changes

**Comment by smaranjitghose** [comment]
@Anushka-shukla  what is the progress on this?

**Pull request merged**